src/code-components/RichTextArea/RichTextArea.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo, useRef } from "react";
+import Quill, { Range } from "quill/core";
+import {
+  formatDefaultToolbarConfigs,
+  Toolbar,
+  ToolbarConfigs,
+} from "./formatDefaultToolbarConfigs";
+import dynamic from "next/dynamic";
```

Comment:
Some apps, such as myevaluations-frontend, don't use next.js . I don't think we can rely on next.js utilities such as this one.